build-tests/s390/tumbleweed/test-image-MicroOS/appliance.kiwi

```diff
@@ -0,0 +1,278 @@
+            </bootloader>
+            <systemdisk>
+                <volume name="home"/>
+                <volume name="root"/>
+                <volume name="opt"/>
+                <volume name="srv"/>
+                <volume name="boot/writable"/>
+                <volume name="usr/local"/>
+                <volume name="var" copy_on_write="false"/>
+            </systemdisk>
+            <size unit="G">2</size>
+        </type>
+    </preferences>
+    <preferences profiles="IBM-Cloud-Secure-Execution-z16">
+        <type
+            image="oem"
+            luks="random"
+            luks_pbkdf="pbkdf2"
+            luks_version="luks2"
+            filesystem="btrfs"
+            kernelcmdline="systemd.show_status=yes console=ttyS0,115200 console=tty0 net.ifnames=0 \$ignition_firstboot ignition.platform.id=qemu swiotlb=262144 rd.debug"
+            devicepersistency="by-uuid"
+            btrfs_root_is_snapshot="true"
+            btrfs_root_is_readonly_snapshot="false"
+            btrfs_root_is_subvolume="true"
+            btrfs_quota_groups="true"
+            bootpartition="true"
+            bootfilesystem="ext3"
+            format="qcow2"
+        >
+            <luksformat>
+                <option name="--cipher" value="aes-xts-plain64"/>
+                <option name="--key-size" value="256"/>
+            </luksformat>
+            <oemconfig>
+                <oem-unattended>true</oem-unattended>
+                <oem-resize>true</oem-resize>
+            </oemconfig>
+            <bootloader name="zipl" timeout="10">
+                <securelinux>
+                    <!-- gen2 z16 certificates -->
+                    <hkd_cert name="/var/lib/se-certs/HKD-3932-02967D8.crt"/>
+                    <hkd_cert name="/var/lib/se-certs/HKD-3932-02967F8.crt"/>
+                    <hkd_cert name="/var/lib/se-certs/HKD-3932-0296878.crt"/>
+                    <hkd_sign_cert name="/var/lib/se-certs/ibm-z-host-key-signing-gen2.crt"/>
+                    <hkd_ca_cert name="/var/lib/se-certs/DigiCertCA.crt"/>
+                    <hkd_revocation_list name="/var/lib/se-certs/ibm-z-host-key.crl"/>
+                    <hkd_revocation_list name="/var/lib/se-certs/ibm-z-host-key-gen2.crl"/>
+                    <hkd_revocation_list name="/var/lib/se-certs/DigiCertTrustedG4CodeSigningRSA4096SHA3842021CA1.crl"/>
+                    <hkd_revocation_list name="/var/lib/se-certs/DigiCertTrustedRootG4.crl"/>
+                </securelinux>
+            </bootloader>
+            <systemdisk>
+                <volume name="home"/>
+                <volume name="root"/>
+                <volume name="opt"/>
+                <volume name="srv"/>
+                <volume name="boot/writable"/>
+                <volume name="usr/local"/>
+                <volume name="var" copy_on_write="false"/>
+            </systemdisk>
+            <size unit="G">2</size>
+        </type>
+    </preferences>
+    <preferences profiles="SUSE-Infra">
```

Comment:
You created three secure executon profiles for your testing which is good.  If the profiles are the same expect for HKDs then should there be only two profile examples with comments that explain these will work for on-premises IBM Z and LinuxONE servers as well in IBM Cloud?
- single-hkd
- multi-hkd

Reply:
Hmm, I can't follow you. I created four profile builds, three of them targets secure execution

1. **SUSE-Infra**, that's for our SUSE LinuxONE
2. **IBM-Cloud-Secure-Execution-z16**, that's for the IBM Cloud regions using z16 systems
3. **IBM-Cloud-Secure-Execution-z15**, that's for the IBM Cloud regions using z15 systems

Actually Nicolas told me there are no z15 systems in the IBM cloud, so maybe the z15 build can be dropped but I left it there because it doesn't hurt

Your merging suggestion of these three into two with single-hkd and multi-hkd I do not understand because all of the three secure execution systems needs a distinct set of certificates for hkd and signing and crls and only the CA is the same for all three. So what exactly do you want to achieve here ?

Please also remember, this is for testing in the kiwi space. I doubt these image descriptions will be the ones you are going to use for a product offer. They probably serve as a base but then will have further modifications.